cli/tests/test_git_clone.rs

```diff
@@ -372,6 +372,50 @@ fn test_git_clone_colocate() {
     "#);
 }
 
+#[test]
+fn test_git_clone_tags() {
+    macro_rules! do_test {
+        ($expected:expr, $args:expr) => {
+            let test_env = TestEnvironment::default();
+            let root_dir = test_env.work_dir("");
+            let git_repo_path = test_env.env_root().join("source");
+            let _git_repo = git::init(git_repo_path);
+
+            // Clone an empty repo
+            let output = root_dir.run_jj(
+                [
+                    "git",
+                    "clone",
+                    "source",
+                    "repo",
+                    "--colocate"
+                ]
+                .iter()
+                .chain($args)
+            );
+            insta::assert_snapshot!(output, @r#"
+            ------- stderr -------
+            Fetching into new repo in "$TEST_ENV/repo"
+            Nothing changed.
+            Hint: Running `git clean -xdf` will remove `.jj/`!
+            [EOF]
+            "#);
+
+            let git_repo = git::open(test_env.env_root().join("repo"));
+            let remote = git_repo.find_remote("origin").unwrap();
+            assert_eq!($expected, remote.fetch_tags());
+        }
+    }
```

Comment:
nit: inline this? I don't think `insta` can update snapshots in macro properly. Another option is to use `insta::allow_duplicates!`.

Reply:
Changing this to a for-loop under `insta::allow_duplicates!` seems to work (and I tested that a broken snapshot will be caught for review that way)